lib/wificom/wifi_picow.py

```diff
@@ -66,4 +66,4 @@ def connect(self):
 				ssl_context=ssl.create_default_context(),
 			)
 
-			return mqtt_client
+		return mqtt_client
```

Comment:
I don't think this belongs in this PR, unless the lib update causes it somehow?

Reply:
Hmm, I see it's in "Fix pylint", and the previous commits were failing, but that's odd.

It's not just fixing pylint, it's a logic change with non-obvious consequences.